psi4/src/psi4/cc/ccdensity/get_params.cc

```diff
@@ -51,24 +51,25 @@ namespace psi {
 namespace ccdensity {
 
 void get_params(Options& options) {
-    int errcod, tol;
-    std::string junk;
-
     params.wfn = options.get_str("WFN");
 
-    // junk = options.get_str("REFERENCE");
-    // if(junk == "RHF") params.ref = 0;
-    // else if(junk == "ROHF") params.ref = 1;
-    // else if(junk == "UHF") params.ref = 2;
-    // else {
-    //  printf("Invalid value of input keyword REFERENCE: %s\n", junk.c_str());
-    //  throw PsiException("ccdensity: error", __FILE__, __LINE__);
-    //}
-
     psio_read_entry(PSIF_CC_INFO, "Reference Wavefunction", (char*)&(params.ref), sizeof(int));
 
-    params.onepdm = options.get_bool("ONEPDM");
-    params.onepdm_grid_dump = options.get_bool("ONEPDM_GRID_DUMP");
+    params.onepdm = options.get_bool("OPDM_ONLY");
+    if (options["ONEPDM"].has_changed()) {
+        outfile->Printf("\tWarning! ONEPDM is deprecated and will be removed in 1.7. Use OPDM_ONLY instead.");
+        if (not options["OPDM_ONLY"].has_changed()) {
+            params.onepdm = options.get_bool("ONEPDM");
+        }
+    }
+
+    params.onepdm_grid_dump = options.get_bool("OPDM_GRID_DUMP");
+    if (options["ONEPDM_GRID_DUMP"].has_changed()) {
+        outfile->Printf("\tWarning! ONEPDM_GRID_DUMP is deprecated and will be removed in 1.7. Use OPDM_GRID_DUMP instead.");
+        if (not options["OPDM_GRID_DUMP"].has_changed()) {
+            params.onepdm = options.get_bool("ONEPDM_GRID_DUMP");
```

Comment:
`params.onepdm_grid_dump` here?

Reply:
Good catch.

My ability to write such stupid mistakes is exactly why I write small PRs if I can get away with it.